### PR TITLE
Service/APT: Return proper parameters in GetLockHandle.

### PR DESCRIPTION
**Pull request by Subv**
Documented some APT functions
This allows applets to boot.

### CODE REVIEW

File: src/core/hle/service/apt/apt.cpp [hunk 1]
```diff
@@ -101,18 +101,19 @@ void NotifyToWait(Service::Interface* self) {
 
 void GetLockHandle(Service::Interface* self) {
     u32* cmd_buff = Kernel::GetCommandBuffer();
-    u32 flags = cmd_buff[1]; // TODO(bunnei): Figure out the purpose of the flag field
+    // Bits [0, 2] are the applet type (System, Library, etc)
```
Comment (yuriks) [inline src/core/hle/service/apt/apt.cpp]: Is this "bit 0 and 2" or "bits 0 to 2"? If the latter, then use "[0:2]" notation which is pretty common for that sort of thing.

**Comment by yuriks** [comment]
LGTM other than the comment

**Comment by Subv** [comment]
Updated

**Comment by Subv** [comment]
Fixed a regression, updated

**Comment by yuriks** [comment]
LGTM (lol forgot to press submit)

**Comment by bunnei** [comment]
LGTM

**Pull request merged**